cfn-resources/go.mod

```diff
@@ -5,19 +5,34 @@ go 1.19
 require (
 	github.com/aws-cloudformation/cloudformation-cli-go-plugin v1.2.0
 	github.com/aws/aws-sdk-go v1.44.230
+	github.com/aws/aws-sdk-go-v2 v1.17.7
```

Comment:
any reason to now include the v2 of the sdk? why not stick to v1? or any plans to then migrate all aws calls to v2?

Reply:
> any plans to then migrate all aws calls to v2
No plan at the moment to migrate to v2.

Are there any reasons why we need v2?

Reply:
SGTM as long as we have a ticket to migrate to v2. Thanks

Reply:
Created https://jira.mongodb.org/browse/INTMDB-726